login.html

```diff
@@ -0,0 +1,106 @@
+                                        d="M2 13.5V7h1v6.5a.5.5 0 0 0 .5.5h9a.5.5 0 0 0 .5-.5V7h1v6.5a1.5 1.5 0 0 1-1.5 1.5h-9A1.5 1.5 0 0 1 2 13.5zm11-11V6l-2-2V2.5a.5.5 0 0 1 .5-.5h1a.5.5 0 0 1 .5.5z" />
+                                    <path fill-rule="evenodd"
+                                        d="M7.293 1.5a1 1 0 0 1 1.414 0l6.647 6.646a.5.5 0 0 1-.708.708L8 2.207 1.354 8.854a.5.5 0 1 1-.708-.708L7.293 1.5z" />
+                                </svg>
+                                Home
+                            </a>
+                        </li>
+                        <li class="nav-item px-lg-3">
+                            <a href="profile.html" class="nav-link">
+                                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" class="bi bi-person">
+                                    <path
+                                        d="M8 8a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm2-3a2 2 0 1 1-4 0 2 2 0 0 1 4 0zm4 8c0 1-1 1-1 1H3s-1 0-1-1 1-4 6-4 6 3 6 4zm-1-.004c-.001-.246-.154-.986-.832-1.664C11.516 10.68 10.289 10 8 10c-2.29 0-3.516.68-4.168 1.332-.678.678-.83 1.418-.832 1.664h10z" />
+                                </svg>
+                                Profile
+                            </a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="login.html" class="nav-link"><button class="btn btn-primary">Login</button></a>
+                        </li>
+                    </ul>
+                </div>
+            </nav>
+        </div>
+    </header>
+    <main>
+        <section class="my-5">
+            <div class="container">
+                <div class="row justify-content-center">
+                    <div class="col-lg-5 bg-white p-4 mb-5 rounded shadow-sm">
+                        <h1>Login</h1>
+                        <div class="d-flex">
+                            <p class="text-muted mr-1">Do not have an account?</p>
+                            <a href="#" class="login-link">Sign up</a>
+                        </div>
+                        <form action="profile.html">
+                            <div class="my-3">
+                                <label for="email" class="form-label">Email address</label>
+                                <input type="email" class="form-control" id="email" placeholder="name@example.com"
+                                    required />
+                            </div>
+                            <div class="mt-4">
+                                <label for="password" class="form-label">Password</label>
+                                <input type="password" class="form-control" id="password"
+                                    placeholder="Enter your password" required />
+                            </div>
+                            <div class="d-flex justify-content-between mt-3">
+                                <div>
```

Comment:
```suggestion
                                <div class="form-check">
```

Look into this: https://getbootstrap.com/docs/5.2/forms/checks-radios/ and this:  https://getbootstrap.com/docs/5.2/forms/input-group/#checkboxes-and-radios